tests/console/login.go

```diff
@@ -139,10 +139,15 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 		&expect.BSnd{S: "sudo su\n"},
 		&expect.BExp{R: PromptExpression},
 	})
-	res, err := expecter.ExpectBatch(b, 3*time.Minute)
-	if err != nil {
-		log.DefaultLogger().Object(vmi).Infof("Login: %+v", res)
-		return err
+	res, err := expecter.ExpectBatch(b, 2*time.Minute)
+	if err != nil {
+		log.DefaultLogger().Object(vmi).Reason(err).Errorf("Login attempt failed: %+v", res)
```

Comment:
Just a question: Is the printed info now different that it is now changed to `Errorf`?

Reply:
> commit message typo: `wating`

fixed.

Reply:
you will only see `level=error` instead of `level=info` and I also print the error itself too, since not every error is necessarily related to not-matching expressions.